source/administration/config-settings.rst

```diff
@@ -2189,7 +2189,8 @@ ID Attribute
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The attribute in the AD/LDAP server used as a unique identifier in Mattermost. It should be an AD/LDAP attribute with a value that does not change.
 
-If a user's ID Attribute changes, it will create a new Mattermost account unassociated with their old one.
+If a user's ID Attribute changes, it will create a new Mattermost account unassociated with their old one. Therefore we recommend setting this to a unique attribute like ``objectGUID`` in Active Directory and ``entryUUID`` in LDAP. 
```

Comment:
```suggestion
If a user's ID Attribute changes, a new Mattermost account (unassociated with the previous one) is created. To prevent this, it's recommended that a unique attribute such as ``objectGUID`` in Active Directory and ``entryUUID`` in LDAP be used instead. 
```